modules/test/playwright/tests/frontend-data-set-views-web/fields.spec.ts

```diff
@@ -12,12 +12,14 @@ import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
 import {fdsFragmentPageTest} from './fixtures/fdsFragmentPageTest';
 import {fieldsPageTest} from './fixtures/fieldsPageTest';
 import {viewsPageTest} from './fixtures/viewsPageTest';
+import {DEFAULT_LABEL} from './utils/constants';
 
 export const test = mergeTests(
 	applicationsMenuPageTest,
 	dataSetsPageTest,
 	fdsFragmentPageTest,
 	featureFlagsTest({
+		'LPD-10735': false,
```

Comment:
This is temporary. What we should do is rewrite Fields tests to use new UI. I can do this in a followup.